src/locales/ko/challenges.ts

```diff
@@ -23,4 +25,8 @@ export const challenges: TranslationEntries = {
     "desc_default": "선택한 타입의 포켓몬만 사용할 수 있습니다."
     //type in pokemon-info
   },
+  "eeveeOnly": {
+    "name": "이브이 모드",
+    "desc": "이 챌린지는 이브이만 사용할 수 있습니다.",
```

Comment:
```suggestion
    "desc": "이 챌린지는 이브이와 이브이의 진화체만 사용할 수 있습니다.",
```
changed korean translation for detail (CC : @sodaMelon)